### PR TITLE
[REF] specifiy order on selections

### PR DESCRIPTION
**Pull request by mart-e**

### CODE REVIEW

**Comment by mart-e** [comment]
cc @tde-banana-odoo using `selection_add` seems the intended behaviour as the onchange makes a reference to an action value from the binding

https://github.com/odoo/odoo/blob/e28d3aa4d3851d72deba08c236072d002f83216a/addons/crm/wizard/crm_lead_to_opportunity.py#L183-L186

File: odoo/fields.py [hunk 1]
```diff
@@ -2055,8 +2085,8 @@ def _selection_modules(self, model):
                     for value, label in field.args['selection']:
                         value_modules[value].add(module)
             if 'selection_add' in field.args:
-                for value, label in field.args['selection_add']:
-                    value_modules[value].add(module)
+                for selection in field.args['selection_add']:
```
Comment (rco-odoo) [inline odoo/fields.py]: variable name: `selection` → `value_label`

File: odoo/fields.py [hunk 2]
```diff
@@ -2055,8 +2085,8 @@ def _selection_modules(self, model):
...
+                    value_modules[selection[0]].add(module)
```
Comment (rco-odoo) [inline odoo/fields.py]: add an `if len(value_label) > 1` for that statement

File: addons/sale_crm/wizard/crm_opportunity_to_quotation.py [hunk 3]
```diff
@@ -24,10 +24,7 @@ def default_get(self, fields):
             result['lead_id'] = active_id
         return result
 
-    action = fields.Selection([
-        ('create', 'Create a new customer'),
-        ('exist', 'Use an existing customer')
-    ], 'Quotation Customer', required=True)
+    action = fields.Selection(string='Quotation Customer')
```
Comment (rco-odoo) [inline addons/sale_crm/wizard/crm_opportunity_to_quotation.py]: Make sure this one is correct.

**Comment by rco-odoo** [comment]
You can also squash the 2nd and 3rd commit.

**Comment by mart-e** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**